pkg/delete/delete_test.go

```diff
@@ -1030,8 +1049,106 @@ func TestDeleteClassicKeyUserActionsWeb(t *testing.T) {
+
+		entriesToDelete := delete.DeleteEntries{given.Type: {given}}
+		err := delete.Configs(context.TODO(), delete.ClientSet{Documents: c}, api.NewAPIs(), automationTypes, entriesToDelete)
+		assert.NoError(t, err)
+	})
+
+	t.Run("config declared via coordinate doesn't exists - no error", func(t *testing.T) {
+		given := pointer.DeletePointer{
+			Type:       "document",
+			Identifier: "monaco_identifier",
+		}
+
+		externalID, _ := idutils.GenerateExternalIDForDocument(given.AsCoordinate())
+		c := client.NewMockDocumentClient(gomock.NewController(t))
+		c.EXPECT().List(gomock.Any(), gomock.Eq(fmt.Sprintf("externalId=='%s'", externalID))).
+			Times(1).
+			Return(documents.ListResponse{Responses: []documents.Response{}}, nil)
+
+		entriesToDelete := delete.DeleteEntries{given.Type: {given}}
+		err := delete.Configs(context.TODO(), delete.ClientSet{Documents: c}, api.NewAPIs(), automationTypes, entriesToDelete)
+		assert.NoError(t, err)
+	})
+
+	t.Run("config declared via coordinate have multiple match - delete them all", func(t *testing.T) {
+		given := pointer.DeletePointer{
+			Type:       "document",
+			Identifier: "monaco_identifier",
+		}
+
+		externalID, _ := idutils.GenerateExternalIDForDocument(given.AsCoordinate())
+		c := client.NewMockDocumentClient(gomock.NewController(t))
+		c.EXPECT().List(gomock.Any(), gomock.Eq(fmt.Sprintf("externalId=='%s'", externalID))).
+			Times(1).
+			Return(documents.ListResponse{Responses: []documents.Response{{ID: "originObjectID_1"}, {ID: "originObjectID_2"}}}, nil)
+
+		entriesToDelete := delete.DeleteEntries{given.Type: {given}}
+		err := delete.Configs(context.TODO(), delete.ClientSet{Documents: c}, api.NewAPIs(), automationTypes, entriesToDelete)
+		assert.Error(t, err)
+	})
+
+	t.Run("delete via originID", func(t *testing.T) {
+		c := client.NewMockDocumentClient(gomock.NewController(t))
+		c.EXPECT().Delete(gomock.Any(), gomock.Eq("originObjectID")).Times(1)
+
+		entriesToDelete := delete.DeleteEntries{
+			"document": {
+				{
+					Type:           "document",
+					OriginObjectId: "originObjectID",
+				},
+			},
+		}
+		err := delete.Configs(context.TODO(), delete.ClientSet{Documents: c}, api.NewAPIs(), automationTypes, entriesToDelete)
+		assert.NoError(t, err)
+	})
+
+	t.Run("config declared via originID doesn't exists - no error", func(t *testing.T) {
+		given := pointer.DeletePointer{
+			Type:           "document",
+			OriginObjectId: "originObjectID",
+		}
+
+		c := client.NewMockDocumentClient(gomock.NewController(t))
+		c.EXPECT().Delete(gomock.Any(), gomock.Eq("originObjectID")).Times(1).Return(documents.Response{}, coreapi.APIError{StatusCode: http.StatusNotFound})
+
+		entriesToDelete := delete.DeleteEntries{given.Type: {given}}
+		err := delete.Configs(context.TODO(), delete.ClientSet{Documents: c}, api.NewAPIs(), automationTypes, entriesToDelete)
 		assert.NoError(t, err)
 	})
 
+	t.Run("error during delete - abort the process", func(t *testing.T) {
```

Comment:
Does an error really cause an abort?

Reply:
If there is an error during the process, the problematic entry is skipped, and an error is reported!
The title of the test has changed